tasmoadmin/pages/upload.php

```diff
@@ -149,6 +158,14 @@
 	}
 }
 elseif (isset($_REQUEST["auto"])) {
+	$useGZIP   = $Config->read("use_gzip_package");
+	if ($useGZIP == 1 || $useGZIP == "1") {
```

Comment:
```suggestion
	if ($useGZIP == 1) {
```

Could be simplified to this since you're not using strict (`===`) equalities

Reply:
Looks great :-D